http-server/src/routes/alin_test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * for example:
+ * http://localhost:3000/alin?time=12:30&volume=5
+ */
+const os = require('os')
+const method = 'GET'
+const route = '/alin'
+var list = [ ];
+const handler = function handler (req, res) {
+  if (!req.query || !req.query.time || !req.query.volume) {
+    res.statusCode = 400
+    res.statusMessage = 'Bad request'
+    return res.end('Bad request')
+  } 
+  //const a = parseFloat(req.query.a)

```

Comment:
Let's keep comments clean, don't comment legacy code, since we got git.
The comments are use to describe "what the function do" if needed.